src/style.css

```diff
@@ -1 +1,76 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin: 0;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  box-sizing: border-box;
+}
+
+.container {
+  margin: 20px 0;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both) to avoid potential margin collapse. Consider using only `margin-top` or `margin-bottom` for the `.container` class.